server/apps/distributed-app/docs/modules/ROOT/pages/configure/extensions.adoc

```diff
@@ -25,4 +25,34 @@ guice.extension.module=com.project.MyServiceModule
 
 Enables to inject MyService into your extensions.
 
+
+*guice.extension.module*: come separated list of fully qualified class name. These classes need to implement Task extension modules.
```

Comment:
` These classes need to implement Task extension modules.`

Please do this in another paragraph:
```
*guice.extension.module*: come separated list of fully qualified class name.

Eg: 



Etensions can rely on the Task manager to supervise long running task execution (progress, await, cancelation, scheduling...). These extensions need to implement Task extension modules. Here is an example of such a class :